osquery/logger/plugins/aws_util.cpp

```diff
@@ -8,15 +8,19 @@
  *
  */
 
+#include <mutex>
```

Comment:
This is the header that defines `std::call_once` that we use below. Is there a reason we should be avoiding it?